src/util/file_reader.ts

```diff
@@ -126,6 +127,7 @@ async function collectLocalPathBehaviors(
     const stat = await fsp.stat(resolvedPath);
 
     if (stat.isFile() && ALLOWED_EXTS.includes(path.extname(resolvedPath))) {
+      logger.info("Custom behavior script added", { filename }, "behavior");
```

Comment:
This may actually be confusing/unhelpful for behaviors downloaded via URL, since we give them an arbitrary filename :/

Reply:
Maybe we add something about where it was downloaded from instead? If its possible to pass in here?
Like displaySource or something?

Reply:
Good call! Changed it to pass in the original filename from the URL for logging purposes